plugins/jobs/src/js/components/form/reducers/JsonReducers.ts

```diff
@@ -0,0 +1,74 @@
+import { deepCopy } from "#SRC/js/utils/Util";
```

Comment:
We usually call JSON reducers JSONReducers.
![screenshot from 2019-03-08 14-20-51](https://user-images.githubusercontent.com/40791275/54028378-733c7380-41ad-11e9-8443-0325f1883f94.png)

Reply:
Since this is separate from the services reducers I don't think it will cause confusion